projects/server-database/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"server-database/types"
+	"strconv"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/joho/godotenv"
+)
+
+func handleImages(w http.ResponseWriter, r *http.Request) {
+	conn, err := setDatabaseConnection()
+	if err != nil {
+		fmt.Fprint(os.Stderr, err)
+	}
+	defer conn.Close(context.Background())
+	images, err := fetchImages(conn)
+	if err != nil {
+		fmt.Fprint(os.Stderr, err)
+	}
+	indent := getIndentParam(r)
+	b, err := json.MarshalIndent(images, "", indent)
+	if err != nil {
+		fmt.Fprint(os.Stderr, err)
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(b)
+}
+
+func getIndentParam(r *http.Request) string {
+	params := r.URL.Query()
+
+	indentSize, err := strconv.Atoi(params.Get("indent"))
```

Comment:
you could distinguish here between cases where indent is not supplied and cases where some string is supplied but it's not parseable as an integer.
An unparseable parameter could be a good reason to return a HTTP 400 code IMO.